src/Morpho.sol

```diff
@@ -472,41 +468,46 @@ contract Morpho is IMorphoStaticTyping {
     /// @dev Accrues interest for the given market `marketParams`.
     /// @dev Assumes that the inputs `marketParams` and `id` match.
     function _accrueInterest(MarketParams memory marketParams, Id id) internal {
-        uint256 elapsed = block.timestamp - market[id].lastUpdate;
-
-        if (elapsed == 0) return;
-
+        // The IRM is always called, useful for stateful IRMs.
         uint256 borrowRate = IIrm(marketParams.irm).borrowRate(marketParams, market[id]);
-        uint256 interest = market[id].totalBorrowAssets.wMulDown(borrowRate.wTaylorCompounded(elapsed));
-        market[id].totalBorrowAssets += interest.toUint128();
-        market[id].totalSupplyAssets += interest.toUint128();
+        // At market creation, elapsed==block.timestamp but totalBorrowAssets==0 so no interest is accrued, as expected.
```

Comment:
This is no longer true with your changes no? Since `market[id].lastUpdate = uint128(block.timestamp);` has been removed at market creation

Reply:
no I think that it's true

Reply:
outdated, see the description of the PR

Reply:
outdated, see the description of the PR